packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -308,10 +297,10 @@ export class ProfilesUtils {
     }
 
     /**
-     * Creates an instance of ProfileInfo and calls `readProfilesFromDisk` to load profiles.
+     * Creates an instance of ProfileInfo with the configured credential manager override.
      * @returns An instance of `ProfileInfo`, or `undefined` if there was an error.
      */
-    public static async getProfileInfo(): Promise<imperative.ProfileInfo> {
+    public static async setupProfileInfo(): Promise<imperative.ProfileInfo> {
```

Comment:
This seems like a good change. I'm curious how many PRs may have conflicts after this 😅 

In theory, not many since this is different from the ProfilesCache.getProfileInfo 😋 

I would love it if this PR was merged ASAP 🙏